homeassistant/components/here_weather/const.py

```diff
@@ -0,0 +1,397 @@
+    "description": {"name": "Description", "unit_of_measurement": None},
+    "skyInfo": {"name": "Sky Info", "unit_of_measurement": None},
+    "skyDescription": {"name": "Sky Description", "unit_of_measurement": None},
+    "temperature": {"name": "Temperature", "unit_of_measurement": "°C"},
+    "temperatureDesc": {"name": "Temperature Description", "unit_of_measurement": None},
+    "comfort": {"name": "Comfort", "unit_of_measurement": "°C"},
+    "highTemperature": {"name": "High Temperature", "unit_of_measurement": "°C"},
+    "lowTemperature": {"name": "Low Temperature", "unit_of_measurement": "°C"},
+    "humidity": {"name": "Humidity", "unit_of_measurement": "%"},
+    "dewPoint": {"name": "Dew Point", "unit_of_measurement": "°C"},
+    "precipitation1H": {
+        "name": "Precipitation Over 1 Hour",
+        "unit_of_measurement": "cm",
+    },
+    "precipitation3H": {
+        "name": "Precipitation Over 3 Hours",
+        "unit_of_measurement": "cm",
+    },
+    "precipitation6H": {
+        "name": "Precipitation Over 6 Hours",
+        "unit_of_measurement": "cm",
+    },
+    "precipitation12H": {
+        "name": "Precipitation Over 12 Hours",
+        "unit_of_measurement": "cm",
+    },
+    "precipitation24H": {
+        "name": "Precipitation Over 24 Hours",
+        "unit_of_measurement": "cm",
+    },
+    "precipitationDesc": {
+        "name": "Precipitation Description",
+        "unit_of_measurement": None,
+    },
+    "airInfo": {"name": "Air Info", "unit_of_measurement": None},
+    "airDescription": {"name": "Air Description", "unit_of_measurement": None},
+    "windSpeed": {"name": "Wind Speed", "unit_of_measurement": "km/h"},
+    "windDirection": {"name": "Wind Direction", "unit_of_measurement": "°"},
+    "windDesc": {"name": "Wind Description", "unit_of_measurement": "cm"},
+    "windDescShort": {"name": "Wind Description Short", "unit_of_measurement": None},
+    "barometerPressure": {"name": "Barometric Pressure", "unit_of_measurement": "mbar"},
+    "barometerTrend": {
+        "name": "Barometric Pressure Trend",
+        "unit_of_measurement": None,
+    },
+    "visibility": {"name": "Visibility", "unit_of_measurement": "km"},
+    "snowCover": {"name": "Snow Cover", "unit_of_measurement": "cm"},
+    "icon": {"name": "Icon", "unit_of_measurement": None},
+    "iconName": {"name": "Icon Name", "unit_of_measurement": None},
+    "iconLink": {"name": "Icon Link", "unit_of_measurement": None},
+    "ageMinutes": {"name": "Age In Minutes", "unit_of_measurement": "min"},
+    "activeAlerts": {"name": "Active Alerts", "unit_of_measurement": None},
+    "country": {"name": "Country", "unit_of_measurement": None},
+    "state": {"name": "State", "unit_of_measurement": None},
+    "city": {"name": "City", "unit_of_measurement": None},
+    "latitude": {"name": "Latitude", "unit_of_measurement": None},
+    "longitude": {"name": "Longitude", "unit_of_measurement": None},
```

Comment:
These are configuration data and shouldn't be in the state machine.

Reply:
Removed